src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -715,4 +715,45 @@
       it { is_expected.to redirect_to(:back) }
     end
   end
+
+  describe 'before_filter set_maintained_project' do
+    before do
+      login user
+      user.home_project.update(kind: 'maintenance')
+    end
+
+    it "#remove_maintained_project raise excepction with invalid maintained project" do
+      expected = expect do
+        post :remove_maintained_project, project: user.home_project, maintained_project: "invalid"
+      end
+      expected.to raise_exception ActiveRecord::RecordNotFound
```

Comment:
It doesn't work. As it is said in the [raise_error examples](http://www.relishapp.com/rspec/rspec-expectations/v/3-2/docs/built-in-matchers/raise-error-matcher):

> raise_error matcher specifies that a block of code raises an error

So without the block the `post` fails before reaching the expect statement.